e2e_tests/database/test_database.json

```diff
@@ -150,16 +150,26 @@
   "fields": {
     "city": 1,
     "cap_exists": true,
+    "cap_exists_rationale": "Ist hier abrufbar:  [Link](https://monitoring.localzero.net/)",
     "target_date_exists": true,
+    "target_date_exists_rationale": "Das Zieljahr ist 2030.",
     "based_on_remaining_co2_budget": true,
+    "based_on_remaining_co2_budget_rationale": "",
     "sectors_of_climate_vision_used": false,
+    "sectors_of_climate_vision_used_rationale": "",
     "scenario_for_climate_neutrality_till_2035_exists": false,
+    "scenario_for_climate_neutrality_till_2035_exists_rationale": "",
     "scenario_for_business_as_usual_exists": false,
+    "scenario_for_business_as_usual_exists_rationale": "Trendszenario ist in Arbeit.",
     "annual_costs_are_specified": false,
+    "annual_costs_are_specified_rationale": "",
     "tasks_are_planned_yearly": false,
+    "tasks_are_planned_yearly_rationale": "",
     "tasks_have_responsible_entity": false,
+    "tasks_have_responsible_entity_rationale": "",
     "annual_reduction_of_emissions_can_be_predicted": false,
```

Comment:
The `_rationale` is missing, here. (Not sure, if I will be able to finish the review today... Someone else may take over.)